### ISSUE TITLE
Why did I get 37.5 AP with "reppoints_minmax_r50_fpn_1x.py" on coco dataset?

### ISSUE THREAD

**Issue by yangli18** [microsoft/RepPoints#3]
Thanks for sharing your work!
I used the following command to train the model on coco dataset. 
```
./mmdetection/tools/dist_train.sh ./configs/reppoints_minmax_r50_fpn_1x.py 8 --validate
```
I have tried using 4 gpus and 8 gpus for model training, but all the results are about 37.5 AP. I wonder what I got wrong or missed? 
Thanks very much!

**Comment by yangze0930** [comment]
Hi, have you decreased the learning rate to 0.005 when you tried using 4 gpus?

**Comment by yangli18** [comment]
No, I used the same initial learning rate 0.01 when training with 8 gpus and 4 gpus. I will try training on 4 gpus with the decreased learning rate and check the results again. Thanks for your suggestion!

**Comment by yangli18** [comment]
@yangze0930 
Hi, I got `38.0 AP` / `58.6 AP50` / `40.8 AP75` this time (0.005 lr, 4 gpus), and it's still different from `38.2 AP` / `59.7 AP50` / `40.7 AP75` in your paper. Is this a normal deviation?

**Comment by yangze0930** [comment]
@yangli18 Sure. In fact, the minmax method seems not as stable as moment method, as only four points receive localization supervision in the refinement.

**Comment by yangli18** [comment]
@yangze0930 I see. Thanks for explaining!
